src/components/Create/components/pages/PayoutsPage/components/ConvertAmountsModal.tsx

```diff
@@ -88,27 +82,31 @@ export const ConvertAmountsModal = ({
     >
       <section className="mb-8 text-sm text-grey-700 dark:text-slate-200">
         <Trans>
-          To switch to 'Limited' payouts, enter a total amount of payouts to
-          split between your recipients.
-        </Trans>
+          To switch to 'Limited' payouts, enter a total amount to pay out of the
+          treasury to split between your recipients.
+        </Trans>{' '}
+        <ExternalLinkWithIcon href={helpPagePath(`/user/project/#payouts`)}>
+          <Trans>Learn more about payout limits</Trans>
+        </ExternalLinkWithIcon>
       </section>
 
       <label className="text-base font-medium text-black dark:text-slate-100">
-        <Trans>Payout total</Trans>
+        <Trans>Payout total (max.)</Trans>
```

Comment:
Hmm, is 'Payout total' not sufficient?

Reply:
The settings version of this modal from Tim had "(max.)", which I do like. To me, indicates a "not necessarily this much, but the highest possible"

Reply:
maybe Maximum? eh, not important